src/zcbenchmarks.cpp

```diff
@@ -31,6 +31,110 @@
 #include "zcash/IncrementalMerkleTree.hpp"
 
 using namespace libzcash;
+void Shutdownpartial()
+{
+    LogPrintf("%s: In progress...\n", __func__);
+    static CCriticalSection cs_Shutdown;
+    TRY_LOCK(cs_Shutdown, lockShutdown);
+    if (!lockShutdown)
+        return;
+
+    /// Note: Shutdown() must be able to handle cases in which AppInit2() failed part of the way,
+    /// for example if the data directory was found to be locked.
+    /// Be sure that anything that writes files or flushes caches only does this if the respective
+    /// module was initialized.
+   //RenameThread("zcash-shutoff");
+ //   mempool.AddTransactionsUpdated(1);
+
+/*     StopHTTPRPC();
+    StopREST();
+    StopRPC();
+    StopHTTPServer();
+ */
```

Comment:
I can understand why this code has been copied directly from `init.cpp` and then commented to show what is not used. However, IMHO we don't need or want any of the commented-out stuff here, because:

- It overly complicates the benchmark file.
- It only makes sense to leave this here if we ensure that we update it in future when `init.cpp` changes, and given that this is all comments, there's not automated way to ensure it. Thus it will end up bitrotting.

Please remove all commented-out code.

Reply:
Done - added comment about where original method is from